src/descriptors/power_system_structs.json

```diff
@@ -2699,6 +2699,20 @@
           },
           "validation_action": "warn"
         },
+        {
+            "name": "storage_target",
+            "comment": "Storage target at the end of simulation as ratio of storage capacity.",
+            "null_value": "0.0",
+            "data_type": "Float64",
+            "default": "1.0"
```

Comment:
default should probably be 0.0, otherwise we'll probably run into infeasibilities if ppl use the default